lib/linguist/languages.yml

```diff
@@ -3306,6 +3306,36 @@ Lex:
   tm_scope: source.lex
   ace_mode: text
   language_id: 199
+LigoLANG:
+  type: programming
```

Comment:
You probably want to add a `color` else it'll default to a very boring grey.

Reply:
Thanks we are discussing it with the team now. 
I will also probably change the git repo to the official LSP repo (can't at the moment because the directory layout make the import bug)

Reply:
color added